Packages/KeyAppKit/Sources/KeyAppNetworking/JsonRpc/JsonRpcDecoder.swift

```diff
@@ -0,0 +1,24 @@
+import Foundation
+
+public struct JsonRpcDecoder: HTTPResponseDecoder {
```

Comment:
move `HTTPResponseDecoder ` implementation to extension

Reply:
Done